zulipterminal/model.py

```diff
@@ -580,6 +582,30 @@ def update_stream_message(
             stream_name = message.get("display_recipient", None)
             old_topic = message.get("subject", None)
             new_topic = request["topic"]
+            stream_title_markup = (
+                "footer_contrast",
+                [
+                    (f" {stream_name} {STREAM_TOPIC_SEPARATOR} "),
+                ],
+            )
+            stream_name = urwid.Text(stream_title_markup)
+            stream_name.markup = stream_title_markup
```

Comment:
We can greatly simplify the formatted text here to `(style_string, text)`, which makes this much easier to code - and test.

The same applies to the topic text.

Reply:
You can see the result of this in the tests, but you've not simplified all the way in the code itself.